pkg/windows/dockerfile

```diff
@@ -81,6 +81,7 @@ CMD powershell.exe -Command Push-Location C:/temp/hubble; \
   Move-Item 'hubble/PortableGit' -Destination './hubble/dist/hubble/' -Force; \
   Move-Item 'C:/ProgramData/chocolatey/lib/NSSM/tools/nssm.exe' -Destination './hubble/dist/hubble/' -Force; \
   Move-Item 'C:/ProgramData/osquery/osqueryi.exe' -Destination './hubble/dist/hubble/' -Force; \
+  Move-Item 'C:/ProgramData/osquery/osqueryd.exe' -Destination './hubble/dist/hubble/' -Force; \
```

Comment:
Dockerfile changes are not yet finalized, any changes in dockerfiles just reflect what could potentially change.